pynamodb_attributes/unicode_delimited_tuple.py

```diff
@@ -44,12 +45,15 @@ def __init__(
         self.delimiter = delimiter
 
     def deserialize(self, value: str) -> T:
-        fields = getattr(self.tuple_type, "_fields", None)
-        field_types = getattr(self.tuple_type, "_field_types", None)
-        if fields and field_types:
-            values = value.split(self.delimiter, maxsplit=len(fields))
+        type_hints = get_type_hints(self.tuple_type)
```

Comment:
Maybe keep calling it `field_types` to clarify what we're dealing with?

Reply:
👍